blueprints/app/files/tsconfig.json

```diff
@@ -26,7 +26,8 @@
       "./node_modules/@ember-data/model/unstable-preview-types",
       "./node_modules/@ember-data/serializer/unstable-preview-types",
       "./node_modules/@ember-data/tracking/unstable-preview-types",
-      "./node_modules/@warp-drive/core-types/unstable-preview-types"
+      "./node_modules/@warp-drive/core-types/unstable-preview-types"<% } else { %>
+      "ember-source/types"<% } %>
```

Comment:
this also fixes a bug where in npm and yarn, none of the types work, since it's possible for there to be no local node_modules

Reply:
see: https://github.com/ember-cli/ember-cli/pull/10612

noting, however, that ember-data doesn't support this, because the `types` exports are currently stripped on publish.
Which is something I'm aiming to address here:
- https://github.com/emberjs/data/pull/9631